packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -687,7 +687,7 @@ async function cloneNativeDirectoriesAsync({
   let copiedPaths: string[] = [];
   let skippedPaths: string[] = [];
   try {
-    await Exp.extractTemplateAppAsync(templateSpec, tempDir, exp);
+    await extractTemplateAppAsync(templateSpec, tempDir, exp);
```

Comment:
Excellent idea, there's no need for this to be in `xdl` since it's only used here. It was a relic from the times when both `exp` and XDE depended on xdl and shared this functionality 😅